content-repo/gendocs.py

```diff
@@ -434,6 +467,8 @@ def process_extra_readme_doc(target_dir: str, prefix: str, readme_file: str, pri
         name = yml_data['title']
         file_id = yml_data.get('id') or normalize_id(name)
         desc = yml_data.get('description')
+        from_version = yml_data.get('fromversion')
```

Comment:
are we sure that all yml files are valid and containing the from/to version definition?

Reply:
From the [schemas](https://github.com/demisto/demisto-sdk/tree/master/demisto_sdk/commands/common/schemas), it looks like only playbooks and only the fromversion is required while toversion or integrations and scripts no

I'll add by default the from version to 0 and to version to 9999 is this good enough?